public/pages/CreateTrigger/containers/DefineCompositeLevelTrigger/TriggerNotificationsContent.js

```diff
@@ -49,7 +49,7 @@ const TriggerNotificationsContent = ({
       name: selectedOptions[0]?.label,
       subject_template: {
         lang: 'mustache',
-        source: 'Monitor {{ctx.monitor.name}} triggered an alert {{ctx.trigger.name}}',
+        source: 'Monitor {{ctx.monitorV2.name}} triggered an alert {{ctx.trigger.name}}',
```

Comment:
Could you confirm this template was supposed to be changed? The file path suggests this is a composite monitor asset, and composite monitors wouldn't use v2 naming patterns, right?